doc/manual/manual.tex

```diff
@@ -10036,6 +10036,49 @@ \subsection{Extending the basic solver}
 \cite{BHK07,BK99m}.}
 
 
+\subsection{Testing \aspect}
+\label{sec:testing}
+
+\aspect{} makes use of a large suite of tests to ensure correct behavior.
+
+\subsubsection{Running tests}
+\label{sec:running_tests}
+
+In order to run the tests, it is necessary to have either Diff or Numdiff to
+compare the results to the known good case. While it is possible to use Diff,
```

Comment:
Maybe add that diff is installed on most Linux systems already and numdiff is usually available as package, so that it is not a severe limitation. Maybe also mention that we use ctest for the testsuite, which should be available on all systems that compiled ASPECT, because it comes with cmake.